lib/srv/desktop/windows_server.go

```diff
@@ -306,7 +303,7 @@ func NewWindowsService(cfg WindowsServiceConfig) (*WindowsService, error) {
 	// authenticate with LDAP when the LDAP server name is not correct
 	// in the certificate).
 	if cfg.LDAPConfig.CA != nil && cfg.LDAPConfig.InsecureSkipVerify {
-		cfg.Log.Warn("LDAP configuration specifies both a CA certificate and insecure_skip_verify." +
+		cfg.Logger.WarnContext(context.Background(), "LDAP configuration specifies both a CA certificate and insecure_skip_verify."+
```

Comment:
```suggestion
		cfg.Logger.Warn("LDAP configuration specifies both a CA certificate and insecure_skip_verify."+
```

It looks like you can use the non-`Context` version of these wherever you're doing a `cfg.Logger.<Level>Context(context.Background(), ...`

<img width="584" alt="image" src="https://github.com/gravitational/teleport/assets/13578537/fa361624-c6bd-4b74-a3e8-aef2a5bac6e7">

Reply:
We have a lint rule that prevents the use of the non-context versions. The context provides the ability to log extra request-scoped values.